src/main/java/com/google/research/bleth/servlets/ListStrategiesServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.research.bleth.servlets;
+
+import com.google.appengine.repackaged.com.google.gson.Gson;
```

Comment:
What is the difference between this and com.google.gson.Gson?

Reply:
The two methods of importing Gson require different maven dependencies. We have used the other method on the branch which didn't end up merged to the master branch (`omermadmon-gson-datastore`), that's where the confusion comes from.

Also, it seems like com.google.appengine.repackaged.com.google.gson.Gson is less recommended, so I will modify it. Thanks.